scripts/build

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Stop on any error
+set -e
+
+script_dirirectory="$( cd "$( dirname "$0" )" && pwd )"
+project_dirirectory=$script_dirirectory/..
+
+# Specify the Docker image name
+image_name='mgmt/mgmt'
```

Comment:
It's o for me to rename it to `purpleidea/mgmt`. I would ask you to create an automated build from this repo.